wallet/wallet-controller/src/lib.rs

```diff
@@ -403,6 +410,99 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static, W: WalletEvents> Controll
             .map_err(ControllerError::WalletError)
     }
 
+    pub async fn create_delegation(
+        &mut self,
+        account_index: U31,
+        address: Address,
+        pool_id: PoolId,
+    ) -> Result<(DelegationId, TxStatus), ControllerError<T>> {
+        let current_fee_rate = self
+            .rpc_client
+            .mempool_get_fee_rate(5)
```

Comment:
Please move this to a constant and explain what this does

Reply:
moved and added a comment